### DISCUSSION TITLE
[WP5420 Discussion]: Two references with incomplete DOIs

### DISCUSSION THREAD

**Discussion by larsgw** [wikipathways/wikipathways-help#89] [pathway-discussion]
### Discussion

Two literature references have incomplete DOIs, and no metadata to determine what the full DOI should be. The creator of the pathway or someone with enough confidence should determine which DOI it should be, or remove the reference altogether. See also previous discussion in https://github.com/wikipathways/wikipathways-database/issues/51.

1. `10.1016/j.jphs.2022.08.00`: in _Journal of Pharmacological Sciences_, missing one digit
2. `10.15586/codonpublications.parkinsonsdisease.2018.ch`: one of the chapters of [_Parkinson's Disease: Pathogenesis and Clinical Aspects_](https://exonpublications.com/index.php/exon/issue/view/9), possibly [Chapter 7: _Pharmacological Treatment of Parkinson’s Disease_](https://exonpublications.com/index.php/exon/article/view/192/350)

### Type of revision

- [ ] Major
- [X] Minor
- [ ] Deletion

**Comment by khanspers** [comment]
@alexmadsen1: Can you take a look at the incomplete DOI's in WP5420?

**Reply by alexmadsen1** [reply]
Is there any chance I can get the node ID or genes?

**Reply by larsgw** [reply]
`10.1016/j.jphs.2022.08.00` is used for a number of nodes, including those of `PCBD1` and `PCBD2`. `10.15586/codonpublications.parkinsonsdisease.2018.ch` is used for a number of metabolites and `COMT`. Or do you need all the node IDs?

**Comment by alexmadsen1** [comment]
Thank you!

**Comment by khanspers** [comment]
Fixed.
